package.json

```
@@ -58,7 +58,7 @@
     "nmr-correlation": "2.3.3"
   },
   "dependencies": {
-    "@atlaskit/pragmatic-drag-and-drop": "^1.7.0",
+    "@atlaskit/pragmatic-drag-and-drop": "^1.6.1",
```

```suggestion
    "@atlaskit/pragmatic-drag-and-drop": "~1.6.1",
```

`^` allows an update to 1.7+